### PR TITLE
Bump containerd to latest upstream

### PR DESCRIPTION
**Pull request by vigh-m**
**Description of changes:**
 Updates containerd and runc to latest upstream. This will resolve [CVE-2023-47108](https://nvd.nist.gov/vuln/detail/CVE-2023-47108)

```
containerd: update to v1.7.20
runc: update to v1.1.13
```

**Testing done:**

- [x] aws-k8s-129 variants tests on both aarch64 and x86_64 architectures. 
```
 NAME                                 TYPE         STATE           PASSED     FAILED     SKIPPED   BUILD ID           LAST UPDATE
 aarch64-aws-k8s-129-conformance      Test         passed             392          0        7020   edc92b35-dirty     2024-07-18T23:38:25Z
 aarch64-aws-k8s-129-quick            Test         passed               5          0        7407   edc92b35-dirty     2024-07-18T21:48:15Z
 x86-64-aws-k8s-129-conformance       Test         passed             392          0        7020   edc92b35-dirty     2024-07-18T23:42:31Z
 x86-64-aws-k8s-129-quick             Test         passed               5          0        7407   edc92b35-dirty     2024-07-18T21:50:52Z
 aarch64-aws-k8s-129                  Resource     completed                                       edc92b35-dirty     2024-07-18T21:46:10Z
 x86-64-aws-k8s-129                   Resource     completed                                       edc92b35-dirty     2024-07-18T21:48:10Z
```

- Tested host-ctr changes by validating that the admin and control containers are up and running
    - Also tested `host-ctr` binary by pulling, running, and cleaning up images.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by vigh-m** [comment]
Updated `host-ctr` containerd dependency and added BRSA advisory info to `advisory/staging`

File: advisories/staging/BRSA-zqgip7w0.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[advisory]
```
Comment (ginglis13) [inline advisories/staging/BRSA-zqgip7w0.toml:1]: nit: could you update this commit message to be more specific? maybe "stage BRSA for containerd CVE-2024-47108"

**Comment by vigh-m** [comment]
Update advisory language and commit message

**Review by ginglis13 — APPROVED** [review]

**Review by larvacea — APPROVED** [review]
I downloaded the tar files and verified the corresponding sha512sum values.

**Review by koooosh — APPROVED** [review]

**Pull request merged**